google/ggrc-core
# GGRC-8030/GGRC-8005 There is no ability to save checked checkboxes Assessments

# Issue description
There is no ability to save checked checkboxes Assessments. 
_It is ticket related with GGRC-8005 and this solution fix it._

# Steps to test changes 
1. Open any Assessment e.g https://ggrc-uat.googleplex.com/assessments/2105#!info
2. Open three dots menu and click Edit Assessment
3. Tick Notify creators, Notify assignees, Notify verifiers checkboxes
4. Save changes
5. Open Edit popup again
6. Review the state of checkboxes
**Actual**: Notify creators, Notify assignees, Notify verifiers checkboxes are not checked
**Expected**: Notify creators, Notify assignees, Notify verifiers checkboxes are checked

# Solution description
Component "checkbox-to-list" was replaced by  "assessment-notifications". Templates inside "checkbox-to-list" tag was moved to "assessment-notification" component template. Interaction with recipients was changed.

# Sanity checklist 

# PR Review checklist
